guides/v3.7.0/templates/input-helpers.md

```diff
@@ -45,13 +45,24 @@ current context.
 
 ## Actions
 
-To dispatch an action on specific events, such as `enter` or `key-press`, use the following
+To dispatch an action on specific events such as `key-press`, use the following
 
 ```handlebars
 {{input value=this.firstName key-press=(action "updateFirstName")}}
 ```
 
-[Event Names](https://www.emberjs.com/api/ember/release/classes/Component) must be dasherized.
+The following event types are supported (dasherised format):
+
+ * `enter`
+ * `insert-newline`
+ * `escape-press`
+ * `focus-in`
+ * `focus-out`
+ * `key-press`
+ * `key-up`
+
+
+More [events types](https://www.emberjs.com/api/ember/release/classes/Component) are also supported but these events need to be written in camelcase format, such `mouseEnter`. Note, there are events of the same type in both the list above and linked. Event names listed above must be dasherised. Additional work is performed on these events.
```

Comment:
You can also directly link to the list of event names by using the header title's id in the url, e.g.:

```suggestion
More [events types](https://www.emberjs.com/api/ember/release/classes/Component#event-names) are also supported but these events need to be written in camelcase format, such `mouseEnter`. Note, there are events of the same type in both the list above and linked. Event names listed above must be dasherised. Additional work is performed on these events.
```

Reply:
Will update these links 👍